website/guides/lyc_timing.md

```diff
@@ -10,322 +10,795 @@ head:
+            type: Number,
+            default: MIN_HBL_LEN,
+            validator(value) {
+                return value >= MIN_HBL_LEN && value <= MAX_HBL_LEN;
+            },
+        },
+        asmFile: String,
+    },
+
+    render() {
+        const MODE_CHANGES = [20, SCANLINE_LEN - this.$props.hblankLength, SCANLINE_LEN];
+
+        const asmFileName = this.$props.asmFile;
+        let asmFile;
+        if (asmFileName) {
+            asmFile = ASM_FILES[asmFileName];
+            if (asmFile === undefined) {
+                throw new SyntaxError(`Unknown ASM file "${asmFileName}" (did you forget to register it in \`ASM_FILES\`?)`)
+            }
+        }
+        let slots = this.$slots.default(); // The slots we'll be working on (shorthand)
+
+        let opsLegend = {}; // Operations with a legend will be registered in this dict
+        let cycles = []; // One entry per row in the table
+
+        let curScanlineCycle = this.$props.offset; // The position inside the current scanline
+        let curInstrCycles = 0; // How many cycles are left of the current instruction
+        for (let slotIdx = 0; slotIdx < slots.length || curInstrCycles !== 0; ) {
+            // Determine this cycle's PPU background color (depending on PPU mode)
+            let modeAt = scanlineCycle => {
+                if (curScanlineCycle < MODE_CHANGES[0]) {
+                    return 2;
+                } else if (curScanlineCycle < MODE_CHANGES[1]) {
+                    return 3;
+                } else {
+                    return 0;
+                }
+            };
+            const ppuColorClass = 'ppu-mode' + modeAt(curScanlineCycle);
+
+            let children = [h('td', { class: ppuColorClass }, '' + curScanlineCycle)];
+
+            // Check if a new instruction begins on this cycle
+            // If so, push a <td> for this instruction
+            if (curInstrCycles == 0) {
+                const slotProps = slots[slotIdx].props;
+                const instrInfo = (function () {
+                    try {
+                        return CPUOp.info(slotProps, asmFile);
+                    } catch (e) {
+                        if (e instanceof SyntaxError && asmFileName && slotProps.line !== undefined) {
+                            e.message += ` (at ${asmFileName}:${slotProps.line})`;
+                        }
+                        throw e;
+                    }
+                })();
+                const instrName = instrInfo.instr;
+
+                let className = 'cpu-' + (instrInfo.class || 'op');
+                if (slotProps.io !== undefined) {
+                    className += ' cpu-io';
+                }
+
+                let props = {
+                    rowspan: instrInfo.cycles,
+                    class: className,
+                    condensed: slotProps.op === 'condensed',
+                };
+                let contents = props.condensed ? h('b', {}, '(...)') : instrName && h('code', {}, instrName);
```

Comment:
![image](https://user-images.githubusercontent.com/14352721/151679752-7f99eb11-9ad0-4b93-85ce-0dc8d88fe3f3.png)

I think a simple `...` fits better, why is that in bold and between parenthesis?

Reply:
Because otherwise it was too easy to miss